.github/workflows/load-test.yml

```diff
@@ -0,0 +1,108 @@
+name: Load Test
+
+on:
+  schedule:
+    - cron: "0 0 * * 1" # Run every Monday at 00:00
+  workflow_dispatch:
+    inputs:
+      test_id:
+        type: string
+        description: "Identifier label for Datadog metrics"
+        default: "billing-relay-load-test"
+      k6_test_path:
+        type: string
+        description: "Path to load test files"
+        default: "test/load/*.js"
+      k6_flags:
+        type: string
+        description: "Additional k6 flags"
+      billing_relay_env_url:
+        type: string
+        description: "URL of the billing relay environment"
+        default: "https://billingrelay.qa.bitwarden.pw"
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  get-secrets:
+    name: Get secrets
+    runs-on: ubuntu-24.04
+    permissions:
+      id-token: write
+    outputs:
+      dd_api_key: ${{ steps.get-kv-secrets.outputs.dd-api-key }}
+      billing_paypal_webhook_key: ${{ steps.get-kv-secrets.outputs.billing-paypal-webhook-key }}
+    steps:
```

Comment:
⛏️ I always like a newline above the steps so they are easier to segment and read.

Reply:
Not fixed 😜

Reply:
Ah! I added it to a different job, it is now fixed. https://github.com/bitwarden/billing-relay/pull/178/commits/dfb9caea9893c4295fe3be81c9ceed1470b48802